webhooks/user_webhook.go

```diff
@@ -0,0 +1,65 @@
+package webhooks
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	controlv1 "github.com/appuio/control-api/apis/v1"
+)
+
+// +kubebuilder:webhook:path=/validate-appuio-io-v1-user,mutating=false,failurePolicy=fail,groups="appuio.io",resources=users,verbs=create;update,versions=v1,name=validate-users.appuio.io,admissionReviewVersions=v1,sideEffects=None
+
+// UserValidator holds context for the validating admission webhook for users.appuio.io
+type UserValidator struct {
+	client  client.Client
+	decoder *admission.Decoder
```

Comment:
Nitpick: IMO we should add permission annotations wherever we need them (Even for these obvious ones)
```suggestion

//+kubebuilder:rbac:groups=appuio.io,resources=organizationmembers,verbs=get

// UserValidator holds context for the validating admission webhook for users.appuio.io
type UserValidator struct {
	client  client.Client
	decoder *admission.Decoder
```